package.json

```diff
@@ -8,10 +8,10 @@
     "build-packages": "pnpm run --recursive --parallel --filter='!./apps/*' --filter='!./packages/sdk-size-test' build",
     "build": "pnpm build-packages && pnpm --filter=@webstudio-is/sdk-size-test build && pnpm --filter=@webstudio-is/builder build",
     "checks": "turbo run checks",
-    "dev": "turbo run dev --parallel",
+    "dev": "pnpm run --parallel --recursive dev",
     "format": "prettier --write \"**/*.{ts,tsx,md}\"",
     "build-storybook": "build-storybook",
-    "storybook": "turbo run storybook:run",
+    "storybook": "pnpm run --parallel --recursive storybook:run",
```

Comment:
don't you loose turbo cache?

Reply:
It's dev mode. There is no cache here. `persistent: true` before was called `cache: false`